contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -86,3 +86,11 @@ There are two options:
     helm upgrade --install -f ...
     ```
     Note: the ClickHouse pod will not be restarted but the `clickhouse-operator` will, no downtime is expected as part of this release.
+
+### Upgrading from 6.0.x
+
+6.1.0 upgrades the Helm dependency chart [`jetstack/cert-manager`](https://github.com/jetstack/cert-manager) from version `1.2.0` to `1.6.1`. This brings some updates to the custom resource definition (CRD).
+
+If you **are not** overriding `cert-manager.installCRDs` by setting it to `false` **there’s nothing you need to do**. Update your Helm release as usual and enjoy your day!
+
+If otherwise you are manually managing the `cert-manager`‘s CRDs, please remember to update the definitions in order to keep everything in sync.
```

Comment:
should we explain how they can change to going from having had set it to false to not manage it manually anymore? (that belongs to 3.x.x update actually - I don't recall why, but for some reason only setting it to false worked for some customer upgrade)

Reply:
Without knowing how people are managing them I don't think we can provide a clean way to move from `installCRDs 
` set to `false` to `true`